instagram-app/src/PostsContainer/ComentSection/CommentSection.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./CommentSection.css";
+import Comment from "./Comment";
+import CommentInput from "./CommentInput";
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
```

Comment:
Good work setting your comments to a class component. For larger applications you probably wouldn't want to do this since the comments will be saved to a database. But this is a good use case for this problem.